### DISCUSSION TITLE
Janet 1.17.2

### DISCUSSION THREAD

**Discussion by bakpakin** [janet-lang/janet#812] [General]
- Remove include of windows.h from janet.h. This caused issues on certain projects.
- Fix formatting in doc-format to better handle special characters in signatures.
- Fix some marshalling bugs.
- Add optional Makefile target to install jpm as well.
- Supervisor channels in threads will no longer include a wasteful copy of the fiber in every
  message across a thread.
- Allow passing a closure to `ev/thread` as well as a whole fiber.
- Allow passing a closure directly to `ev/go` to spawn fibers on the event loop.

<hr /><em>This discussion was created from the release <a href='https://github.com/janet-lang/janet/releases/tag/v1.17.2'>Janet 1.17.2</a>.</em>

**Comment by sogaiu** [comment]
Thank you for the fixes and enhancements!
